### DISCUSSION TITLE
Spotify audio not working.

### DISCUSSION THREAD

**Discussion by Ghufran496** [muxinc/media-chrome#849] [Q&A] [answered]
does anybody know? spotify audio not working it is not even working on sand box whcih they have provided,.
fix plz

**Comment by rehanIrshad** [comment]
"Try troubleshooting your device's audio settings or reinstalling the Spotify app to resolve the issue. If you're looking for alternative options, consider exploring [Spotify Mod APK AN1](https://www.getmodsapk.cc/spotify-premium-apk/) for additional features and functionality."

**Comment by luwes** [comment, marked as answer]
Fixed in 
https://github.com/luwes/spotify-audio-element/commit/24ae09becc4c5ee540009c0d9a86e1c2d6f78358
https://www.media-chrome.org/docs/en/media-elements/spotify-audio
